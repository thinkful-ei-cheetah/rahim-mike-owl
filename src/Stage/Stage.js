import React from 'react';
import './Stage.css';
import store from '../STORE';

export default function Stage(props) {
  return (
    <div className='views-container views-container__with-sidebar'>
      <div>
        <p>{store.participants.name}</p>
      </div>
      <div>
        <img src={store.participants.avatar} />
      </div>
    </div>
  );
}
