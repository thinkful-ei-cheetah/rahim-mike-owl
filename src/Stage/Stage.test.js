import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Stage from './Stage';

describe('Stage component', () => {
  it('renders stage component without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Stage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders UI with data', () => {
    const tree = renderer
      .create(<Stage name={'hello'} avatar={'world'} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
