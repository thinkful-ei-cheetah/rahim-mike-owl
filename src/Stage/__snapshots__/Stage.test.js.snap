// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stage component renders UI with data 1`] = `
<div
  className="views-container views-container__with-sidebar"
>
  <div>
    <p />
  </div>
  <div>
    <img />
  </div>
</div>
`;
